app/client/packages/design-system/widgets/src/components/MultiSelect/src/MultiSelectValue.tsx

```
@@ -0,0 +1,165 @@
+  });
+
+  useLayoutEffect(updateOverflow, [updateOverflow, selectedKeys]);
+
+  return (
+    <Button
+      className={clsx(
+        textInputStyles.input,
+        selectStyles.selectTriggerButton,
+        styles.multiSelectValue,
+      )}
+      data-invalid={Boolean(isInvalid) ? "" : undefined}
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)